PROCESSORS.md

```diff
@@ -1640,6 +1641,35 @@ In the list below, the names of required properties appear in bold. Any other pr
 | success | All files are routed to success |
 
 
+## PostElasticsearch
+
+### Description
+
+An Elasticsearch/Opensearch post processor that uses the Elasticsearch/Opensearch _bulk REST API.
+### Properties
+
+In the list below, the names of required properties appear in bold. Any other properties (not in bold) are considered optional. The table also indicates any default values, and whether a property supports the NiFi Expression Language.
+
+| Name                                           | Default Value | Allowable Values | Description                                                                                                                                                                                                                                                                                               |
+|------------------------------------------------|---------------|------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Action                                         |               |                  | The type of the operation used to index (create, delete, index, update, upsert)<br/>**Supports Expression Language: true**                                                                                                                                                                                |
+| Max Batch Size                                 | 100           |                  | The maximum number of Syslog events to process at a time.                                                                                                                                                                                                                                                 |
```

Comment:
syslog events?
```suggestion
| Max Batch Size                                 | 100           |                  | The maximum number of flow files to process at a time.                                                                                                                                                                                                                                                 |
```

Reply:
oops, nice catch, I've included this in https://github.com/apache/nifi-minifi-cpp/pull/1349/commits/fdb1bf2a34423815cfe2cb395e83dcee3bc2f1cd#diff-fd2410931e7fdc4bf8b3ce23f5f7a27c7aacdf9337320626d86d806448c90b9bR1656